scripts/upload-build

```diff
@@ -33,12 +33,11 @@ if [ -z "$OPENRCT2_ORG_TOKEN" ]; then
     echo -e "\033[0;31mOPENRCT2_ORG_TOKEN not set"
     exit 1
 fi
-curl -m 300 --connect-timeout 5 -o - \
+curl --include -m 300 --connect-timeout 5 -o - \
     --form "key=$OPENRCT2_ORG_TOKEN" \
     --form "fileName=$filename" \
     --form "version=$version" \
     --form "gitHash=$sha1" \
     --form "gitBranch=$branch" \
-    --form "flavourId=$flavourid" \
```

Comment:
Isn't the flavour ID necessary for the server to know where to put it?

Reply:
No, we moved away from this last year.